applications/Liberland_judiciary.md

```diff
@@ -0,0 +1,533 @@
+
+**[DOT Marketplace](https://github.com/WowLabz/dot-marketplace-v2/blob/Phase3_Milestone3/pallets/pallet-tasking/src/lib.rs)**: This proposal presents a general marketplace that could potentially offer judicial and legal services, contingent on market demand. However, its scope is both broader and more specific than our solution. While it is not a specialized judicial solution, it is also too narrowly focused on marketplace and task functions to present a significant competitive threat. Notably, it does not explicitly propose a meta-system to handle disputes arising within the marketplace.
+
+In light of these observations, we believe that our solution does not compete with either the SaaS3 Court or the DOT Marketplace. Instead, we see potential opportunities for collaboration with these initiatives. Our unique approach to providing a comprehensive legal framework on the blockchain sets us apart and positions us to address a distinct set of challenges and needs within the Polkadot Ecosystem.
+
+As of the time of writing, there are no known projects within the Substrate/Polkadot/Kusama ecosystem that offer the same combination of features as the Liberland Blockchain project. We are the first country to run its government (and judiciary) on the blockchain. But besides that, the unique aspect of this project is the integration of a 'court as a smart contract oracle' that enables the execution of traditional contracts on-chain and the introduction of a entity capable of interpreting intent behind actions within the blockchain. The entire world of traditional contracts is currently barred entry into the blockchain world, and we are attempting to forge a key allowing that huge, majority part of the world economy to come into the Polkadot ecosystem.
+
+
+## Team :busts_in_silhouette:
+
+### Team members
+
+- Dorian Stern-Vukotic, Senior Developer
+- Kacper Zuk, Senior Developer
+
+### Contact
+
+- **Contact Name:** Dorian Štern Vukotić
+- **Contact Email:** [dorian.sternvukotic@gmail.com ]
+- **Website:** [Liberland](www.liberland.org)
+
+### Legal Structure
+
+- **Registered Address:** 3/F, Chinachem Tower, 34-37 Connaught Road Central, Hong Kong
+- **Registered Legal Entity:** Liberland Limited
+
+### Team's experience
+
+Our team has extensive experience in developing blockchain solutions. We have been working on the Liberland Blockchain, a layer 1 public blockchain owned by coin and token holders. All team members individually have 7+ years of software engineering experience with 1+ years experience with blockchain technologies, primarily Substrate, Rust and Solidity.
+
+### Team Code Repos
+
+- [Liberland GitHub](https://github.com/liberland/liberland)
+
+### Team LinkedIn Profiles (if available)
+
+- [Dorian Stern-Vukotic](https://www.linkedin.com/in/djstern)
+- [Kacper Zuk](https://www.linkedin.com/in/kacperzuk/)
+
+## Development Status :open_book:
```

Comment:
Do you have any existing code that the judiciary/contracts would interact with? I imagine it's not completely independent of other parts of the system. 

Also, what is the legal framework for the off-chain judiciary? Is it described anywhere, like a constitution?

Reply:
> Do you have any existing code that the judiciary/contracts would interact with?

Yesnt. Our plan is to refactor existing code to use contracts (for example citizenship logic as part of Liberland LLM pallet) instead of making contracts interact with our custom pallets, but that part is out of scope for this grant.

> Also, what is the legal framework for the off-chain judiciary? Is it described anywhere, like a constitution?`

Most of it is here, though after the first election in about 60 days the laws will be ready to be put on-chain, and some of them incorporated directly in the citizenship contract.
[Draft of Liberland law](https://github.com/liberland/laws/tree/Laws-Current)